### PR TITLE
logistics / cleanup for 1.1 release 

### PR DESCRIPTION
**Pull request by hpk42**
- [x] make 1.1 available as PDF 
- [x] update events
- [x] go through FAQ and streamline were neccessary
- [x] update implementation status with balsa and logos (please check!)

### CODE REVIEW

**Comment by hpk42** [comment]
preview is at https://autocrypt.org/_unofficial_unreleased_docs/pdf11/

File: doc/faq.rst [hunk 1]
```diff
@@ -24,7 +24,69 @@ There are a couple of reasons for holding off on this for now:
...
+
+How does Autocrypt differ from PEP?
+---------------------------------------------------------------
+
+Autocrypt and PEP are different approaches to opportunistic e-mail encryption.
+
+The PEP effort predates Autocrypt by a few years and involves a foundation
+and two companies which provide FOSS and paid offerings.  PEP offers the
+GPL-licensed PEP-Engine and adapters which can be integrated into MUAs
+if licensing permits.  Some specification efforts have started but
```
Comment (Valodim) [inline doc/faq.rst]: Specification efforts had not started when we began work on Autocrypt ([first commit](https://pep.foundation/dev/repos/internet-drafts/rev/48f768143a7c) is from december 2016), and so far there has been no official release.

File: doc/faq.rst [hunk 2]
```diff
@@ -33,29 +95,34 @@ Attachments are visible to users of non Autocrypt-compatible MUAs,
...
-requires the MUA to keep the information who announced Autocrypt and
-who they requested keys from.
+We played with capabilities in a previous design. We decided against them because
+they require the MUA to keep the information who announced Autocrypt and who
+they requested keys from and that is complicated for multi-device settings, in particular.
 
 
-Why RSA3072 and 25519 only later?
----------------------------------
+Why also RSA3072 and not only Curve 25519 keys?
```
Comment (Valodim) [inline doc/faq.rst]: Just for the record, this is also answered in a note in the spec itself: https://autocrypt.org/level1.html#openpgp-based-key-data

File: doc/faq.rst [hunk 3]
```diff
@@ -24,7 +24,69 @@ There are a couple of reasons for holding off on this for now:
...
+There are also a few technical differences despite both efforts supporting
+opportunistic key management, for example:
+
+- PEP generates one key per device and encrypts to multiple keys for one
+  recipient. Autocrypt only uses one key per e-mail address, to reduce
+  security concerns and implementation complexity.
+
+- Autocrypt MUAs specifies how to share the encryption setup between different
+  mail apps and this is implemented and reported to work between
+  different Autocrypt-enabled MUAs.  PEP does not offer sharing of the
```
Comment (lefherz) [inline doc/faq.rst]: Could it be that "Autocrypt MUAs specifies" should actually be "Autocrypt specifies", and "MUAs " only slipped in by accident?

Also this sentence could be split up, leaving out the "and". Easier to understand.

**Pull request merged**